### ISSUE TITLE
cross compilation

### ISSUE THREAD

**Issue by anzbert** [jbloit/androidLinkTest#1]
Hey!

I am trying to compile ableton link for android on my macbook and i was wondering what computer you used ? was it linux? because i am struggling over here :)

**Comment by jbloit** [comment]
Hi, I haven't trie to compile this for a while, but it was for sure compiled on a mac. What kind of errors are you getting?

**Comment by anzbert** [comment]
Yes I could compile it easily on macos, just following the instructions on the readme. The cross compilation to android was not as straightforward. The errors mostly highlighted missing dependencies. I can let you know exactly what the output was once I can spend a little more time trying this again.. hopefully on the weekend.

Thx for the help!

**Comment by anzbert** [comment]
I am using the android sdk/ndk with the dart/flutter framework, so there is a lot going on with different technologies. I want to eventually create a flutter plugin with FFI to the Ableton link C bindings. For Android and maybe other platforms eventually.

So far I am already stumbling at the first step haha 😅 and I've spent many hours trying. I am not very adept at cmake though, so there is that learning curve.

Edit: I have successfully compiled link on Mac, Linux and windows without any problems beforehand. That seems to be the easy bit. The android sdk is giving me the problems I think
